f2/src/forms/ScammerDetailsForm.js

```diff
@@ -69,129 +67,139 @@ export const ScammerDetailsFormWrapped = props => {
     props.onSubmit(client, data)
   }
   return (
-    <ApolloConsumer>
-      {client => (
-        <Form
-          onSubmit={() => localSubmit(client)}
-          render={({ handleSubmit }) => (
-            <Stack
-              as="form"
-              onSubmit={handleSubmit}
-              spacing={6}
-              shouldWrapChildren
-            >
-              <Field name="scammerDetails">
-                {props => (
-                  <FormControl>
-                    <FormLabel htmlFor="scammerDetails">
-                      <Text fontWeight="bold">
-                        <Trans id="scammerDetail.summary" />
-                      </Text>
-                    </FormLabel>
-                    <FormHelperText variant="above">
-                      <Text color="blackAlpha.600">
-                        <Trans id="scammerDetail.reminder" />
-                      </Text>
-                    </FormHelperText>
+    <React.Fragment>
+      {false ? ( // mark ids for lingui
+        <div>
+          <Trans id="fileUpload.removed" /> <Trans id="fileUpload.added" />
```

Comment:
Similarly here we're marking the ids here and changing the indentation of the other lines.